src/bevy/simple_insert.rs

```diff
@@ -1,16 +1,16 @@
 use bevy_ecs::prelude::*;
 use cgmath::*;
 
-#[derive(Copy, Clone)]
+#[derive(Copy, Clone, bevy_ecs::component::Component)]
```

Comment:
For an insertion-focused test, I would suggest using Bevy's sparse set storage rather than the default table storage. Ideally there'd be benches for both, but that may be hard to display.

Reply:
For enabling the Sparse Set storage you would add a `#[component(storage = "SparseSet")]` below the Component derive

Reply:
Having bench only for SparseSet based storage wouldn't be fair.
It's a tradeoff with significant iteration speed disadvantage.

Having both would be perfect.

Reply:
Yep, I'm on board there. I'd love to see both storage types for iteration and insertion speed.